src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+toc: true
+docs_area: migrate
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+MOLT LMS (Live Migration Service) is used to perform a [live migration]({% link {{ page.version.version }}/migration-overview.md %}#minimal-downtime) to CockroachDB.
+
+The LMS is a self-hosted, horizontally scalable proxy that routes traffic between an application, a source database, and a target CockroachDB database. You can use the LMS to control which database, as the "source of truth", is serving reads and writes to an application. You can optionally configure the LMS to [shadow production traffic](#shadowing-modes) from the source database and validate the query results on CockroachDB. When you have sufficiently tested your application and are confident with its consistency and performance on CockroachDB, you use the LMS to [perform the cutover](#perform-a-cutover) to CockroachDB.
+
+MOLT LMS is self-hosted on [Kubernetes](https://kubernetes.io/) and [configured using Helm](#configuration). At a high level, the LMS consists of the following:
+
+- A number of proxy [instances](#lms-instances) (running in separate Kubernetes pods) across which application traffic is distributed and routed to the source and target databases.
+- An "orchestrator" service (running in a single Kubernetes pod) that coordinates the proxy instances and sends the cutover commands.
+
+This page describes how to [install](#installation), [configure](#configuration), [secure](#security), and [use the LMS](#molt-lms-cli) to perform a live migration. {% comment %}For more information, see [Migration Strategy: Live Migration]({% link {{ page.version.version }}/migration-strategy-live-migration.md %}).{% endcomment %}
+
+## Terminology
+
+- A *live migration* involves maintaining two production databases (a source and target database) and replicating data between them until a final cutover.
+- The *source of truth* is the database that serves reads and writes to the application during a live migration. A cutover switches the source of truth.
+- *Shadowing* is the replication of writes from the source of truth to the target database. The LMS supports multiple [shadowing modes](#shadowing-modes).
+
+## Requirements
+
+- [Kubernetes](https://kubernetes.io/) cluster
+- [Helm](https://helm.sh/docs/intro/install/) package manager for Kubernetes
+
+#### Supported databases
+
+- [PostgreSQL]({% link {{ page.version.version }}/migrate-from-postgres.md %}) (source)
+- [MySQL]({% link {{ page.version.version }}/migrate-from-mysql.md %}) (source)
+- CockroachDB (target)
+
+## Installation
+
+The LMS definitions are located in the [`molt` repository](https://molt.cockroachdb.com/):
+
+- [`molt-lms` Helm chart](https://molt.cockroachdb.com/charts/lms/Chart.yaml)
+- [`molt-lms` values file](https://molt.cockroachdb.com/charts/lms/values.yaml) for [configuration](#configuration)
```

Comment:
I'm not sure if this is the right thing to link to unless we are creating redirect with this url to the actual github repo. The molt.cockroachdb.com item is a packaged chart not like what you would view in github directly. I think this maybe more useful to point to the exact github file. cc @ryanluu12345

Reply:
OK, I had the direct GitHub links originally. I'm not clear on what exactly will be at `molt.cockroachdb.com` and where.

Reply:
So at `molt.cockcroachdb.com/lms/cli/` will live all the binaries. At `molt.cockroachdb.com/lms/charts` will live the chart that someone can use to `helm repo add`. But that will be the fully packaged helm chart and not the raw `.yaml` themselves. @Jeremyyang920 is correct that the raw charts will live in the Github itself.